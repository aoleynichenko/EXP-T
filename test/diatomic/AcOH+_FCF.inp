#
# AcOH+.dat
# =========
#
# calculation of Frank-Condon factors for the (1)1/2-(1)3/2 transition in AcOH+
# potential curves were calculated by the relativistic FS-CCSD(0h1p) method,
# 
# for more details, see:
# A. Oleynichenko, L. V. Skripnikov, A. V. Zaitsevskii, V. V. Flambaum
# Laser-coolable AcOH+ ion for CP-violation searches (2022)
# arXiv:2112.02307 [physics.atom-ph]
#
# A. Oleynichenko, 9 Jan 2022
#

#          Ac         OH
masses 227.028000 17.003000

rotlevels 0 0
viblevels 0 5

potential atomic atomic
0.340150702E+01  -0.451045595E+03   1.0  -0.451004746E+03
0.349599333E+01  -0.451064998E+03   1.0  -0.451025470E+03
0.359047964E+01  -0.451079018E+03   1.0  -0.451040834E+03
0.368496594E+01  -0.451088543E+03   1.0  -0.451051696E+03
0.377945225E+01  -0.451094322E+03   1.0  -0.451058791E+03
0.387393856E+01  -0.451096994E+03   1.0  -0.451062746E+03
0.396842486E+01  -0.451097103E+03   1.0  -0.451064095E+03
0.406291117E+01  -0.451095111E+03   1.0  -0.451063297E+03
0.415739747E+01  -0.451091413E+03   1.0  -0.451060740E+03
0.425188378E+01  -0.451086341E+03   1.0  -0.451056758E+03
0.434637009E+01  -0.451080177E+03   1.0  -0.451051631E+03
0.444085639E+01  -0.451073161E+03   1.0  -0.451045598E+03
0.453534270E+01  -0.451065491E+03   1.0  -0.451038858E+03
end
