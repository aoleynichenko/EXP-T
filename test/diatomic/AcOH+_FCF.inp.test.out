
	**********************************************************************************
	**                                                                              **
	**           program for rotational-vibrational levels and properties           **
	**                              of diatomic molecules                           **
	**                                                                              **
	**                              2022 A. Oleynichenko                            **
	**                       compiled Jan 11 2022 13:23:55                          **
	**                                                                              **
	**********************************************************************************

 > started at Tue Jan 11 13:24:44 2022

 > echo of the input file

   #
   # AcOH+.dat
   # =========
   #
   # calculation of Frank-Condon factors for the (1)1/2-(1)3/2 transition in AcOH+
   # potential curves were calculated by the relativistic FS-CCSD(0h1p) method,
   # 
   # for more details, see:
   # A. Oleynichenko, L. V. Skripnikov, A. V. Zaitsevskii, V. V. Flambaum
   # Laser-coolable AcOH+ ion for CP-violation searches (2022)
   # arXiv:2112.02307 [physics.atom-ph]
   #
   # A. Oleynichenko, 9 Jan 2022
   #
   
   #          Ac         OH
   masses 227.028000 17.003000
   
   rotlevels 0 0
   viblevels 0 5
   
   potential atomic atomic
   0.340150702E+01  -0.451045595E+03   1.0  -0.451004746E+03
   0.349599333E+01  -0.451064998E+03   1.0  -0.451025470E+03
   0.359047964E+01  -0.451079018E+03   1.0  -0.451040834E+03
   0.368496594E+01  -0.451088543E+03   1.0  -0.451051696E+03
   0.377945225E+01  -0.451094322E+03   1.0  -0.451058791E+03
   0.387393856E+01  -0.451096994E+03   1.0  -0.451062746E+03
   0.396842486E+01  -0.451097103E+03   1.0  -0.451064095E+03
   0.406291117E+01  -0.451095111E+03   1.0  -0.451063297E+03
   0.415739747E+01  -0.451091413E+03   1.0  -0.451060740E+03
   0.425188378E+01  -0.451086341E+03   1.0  -0.451056758E+03
   0.434637009E+01  -0.451080177E+03   1.0  -0.451051631E+03
   0.444085639E+01  -0.451073161E+03   1.0  -0.451045598E+03
   0.453534270E+01  -0.451065491E+03   1.0  -0.451038858E+03
   end

 > input data:

   reduced mass        : 15.81830622 (amu) = 28835.00827101 (electron masses)
   vibrational levels  : from 0 to 5
   rotational levels   : from 0 to 0
   elec transition     : yes
   write wavefun files : no
   grid size           : 300 points
   solver (integrator) : numerov
   mapping of r grid   : identity = off

           r [a.u.]          U1(r) [a.u.]  
         3.401507020000   -451.045595000000
         3.495993330000   -451.064998000000
         3.590479640000   -451.079018000000
         3.684965940000   -451.088543000000
         3.779452250000   -451.094322000000
         3.873938560000   -451.096994000000
         3.968424860000   -451.097103000000
         4.062911170000   -451.095111000000
         4.157397470000   -451.091413000000
         4.251883780000   -451.086341000000
         4.346370090000   -451.080177000000
         4.440856390000   -451.073161000000
         4.535342700000   -451.065491000000

           r [a.u.]          U2(r) [a.u.]  
         3.401507020000   -451.004746000000
         3.495993330000   -451.025470000000
         3.590479640000   -451.040834000000
         3.684965940000   -451.051696000000
         3.779452250000   -451.058791000000
         3.873938560000   -451.062746000000
         3.968424860000   -451.064095000000
         4.062911170000   -451.063297000000
         4.157397470000   -451.060740000000
         4.251883780000   -451.056758000000
         4.346370090000   -451.051631000000
         4.440856390000   -451.045598000000
         4.535342700000   -451.038858000000

           r [a.u.]           property(r)  
         3.401507020000      1.000000000000
         3.495993330000      1.000000000000
         3.590479640000      1.000000000000
         3.684965940000      1.000000000000
         3.779452250000      1.000000000000
         3.873938560000      1.000000000000
         3.968424860000      1.000000000000
         4.062911170000      1.000000000000
         4.157397470000      1.000000000000
         4.251883780000      1.000000000000
         4.346370090000      1.000000000000
         4.440856390000      1.000000000000
         4.535342700000      1.000000000000

 > begin ground state potential

 > equilibrium parameters

   e(min) = -451.097340595910 a.u. = -99004422.536207 cm^-1
   re     = 3.9249 a.u. = 2.0770 angstrom

 > analysis in the harmonic oscillator + rigid rotor approximation

   we = 656.6654 cm^-1
   Be = 0.2470 cm^-1

 > analysis in the morse approximation

   (parameters are estimated at the minimum point only)
   alpha = 1.0793 bohr^-1
   De    = 0.110795 a.u. = 24317 cm^-1
   we    = 656.67 cm^-1
   wexe  = 4.43 cm^-1
   vmax  = 73

 > rovibrational energy levels

     J   v       energy, cm^-1        < r >, A        < prop >

@    0   0            327.5900        2.080190        1.000000
@    0   1            979.7793        2.086699        1.000000
@    0   2           1628.1902        2.093255        1.000000
@    0   3           2273.1511        2.099903        1.000000
@    0   4           2914.7633        2.106565        1.000000
@    0   5           3552.8046        2.113172        1.000000

 > begin excited state potential

 > equilibrium parameters

   e(min) = -451.064104876888 a.u.
   re     = 3.9775 a.u. = 2.1048 angstrom
   Te     = 7294.3972 cm^-1

 > analysis in the harmonic oscillator + rigid rotor approximation

   we = 626.1237 cm^-1
   Be = 0.2406 cm^-1

 > analysis in the morse approximation

   (parameters are estimated at the minimum point only)
   alpha = 0.9570 bohr^-1
   De    = 0.128110 a.u. = 28117 cm^-1
   we    = 626.12 cm^-1
   wexe  = 3.49 cm^-1
   vmax  = 89

 > rovibrational energy levels

     J   v       energy, cm^-1        < r >, A        < prop >

@    0   0           7607.4633        2.108194        1.000000
@    0   1           8231.0560        2.114939        1.000000
@    0   2           8851.0894        2.121712        1.000000
@    0   3           9467.5003        2.128507        1.000000
@    0   4          10080.5784        2.135318        1.000000
@    0   5          10691.0647        2.141666        1.000000

 > rovibronic transition matrix elements

(J',v')    ground level
(J'',v'')  excited level

    J' J''  v' v''        e1,cm^-1        e2,cm^-1   delta e,cm^-1            prop        |prop|^2
$    0   0   0   0        327.5900       7607.4633       7279.8734       -0.942947        0.889149
$    0   0   0   1        327.5900       8231.0560       7903.4660        0.320665        0.102826
$    0   0   0   2        327.5900       8851.0894       8523.4994       -0.086871        0.007547
$    0   0   0   3        327.5900       9467.5003       9139.9103        0.021282        0.000453
$    0   0   0   4        327.5900      10080.5784       9752.9884        0.004935        0.000024
$    0   0   0   5        327.5900      10691.0647      10363.4747        0.001111        0.000001
$    0   0   1   0        979.7793       7607.4633       6627.6841        0.325405        0.105888
$    0   0   1   1        979.7793       8231.0560       7251.2767        0.831826        0.691934
$    0   0   1   2        979.7793       8851.0894       7871.3101       -0.423973        0.179754
$    0   0   1   3        979.7793       9467.5003       8487.7210        0.143579        0.020615
$    0   0   1   4        979.7793      10080.5784       9100.7991        0.041076        0.001687
$    0   0   1   5        979.7793      10691.0647       9711.2854        0.010714        0.000115
$    0   0   2   0       1628.1902       7607.4633       5979.2731        0.069678        0.004855
$    0   0   2   1       1628.1902       8231.0560       6602.8657        0.437076        0.191036
$    0   0   2   2       1628.1902       8851.0894       7222.8991        0.726560        0.527889
$    0   0   2   3       1628.1902       9467.5003       7839.3100       -0.484342        0.234587
$    0   0   2   4       1628.1902      10080.5784       8452.3882       -0.193328        0.037376
$    0   0   2   5       1628.1902      10691.0647       9062.8744       -0.062543        0.003912
$    0   0   3   0       2273.1511       7607.4633       5334.3122        0.010337        0.000107
$    0   0   3   1       2273.1511       8231.0560       5957.9049        0.117404        0.013784
$    0   0   3   2       2273.1511       8851.0894       6577.9383        0.507664        0.257723
$    0   0   3   3       2273.1511       9467.5003       7194.3492        0.627243        0.393434
$    0   0   3   4       2273.1511      10080.5784       7807.4273        0.520283        0.270695
$    0   0   3   5       2273.1511      10691.0647       8417.9136        0.237330        0.056326
$    0   0   4   0       2914.7633       7607.4633       4692.7001        0.001070        0.000001
$    0   0   4   1       2914.7633       8231.0560       5316.2927        0.020363        0.000415
$    0   0   4   2       2914.7633       8851.0894       5936.3261        0.161442        0.026064
$    0   0   4   3       2914.7633       9467.5003       6552.7370        0.554908        0.307923
$    0   0   4   4       2914.7633      10080.5784       7165.8151       -0.533865        0.285012
$    0   0   4   5       2914.7633      10691.0647       7776.3014       -0.540216        0.291833
$    0   0   5   0       3552.8046       7607.4633       4054.6587       -0.000079        0.000000
$    0   0   5   1       3552.8046       8231.0560       4678.2514       -0.002412        0.000006
$    0   0   5   2       3552.8046       8851.0894       5298.2848       -0.031734        0.001007
$    0   0   5   3       3552.8046       9467.5003       5914.6957       -0.202475        0.040996
$    0   0   5   4       3552.8046      10080.5784       6527.7738        0.586408        0.343874
$    0   0   5   5       3552.8046      10691.0647       7138.2601       -0.446945        0.199760

 > finished at Tue Jan 11 13:24:44 2022

